socketshark/__init__.py

```diff
@@ -126,35 +128,28 @@ async def prepare(self):
 
         Initialize Redis connection and the receiver class.
         """
-        redis_receiver = Receiver(loop=asyncio.get_event_loop())
-        redis_settings = self.config['REDIS']
+        redis_settings = [self.config['REDIS']]
+        if self.config.get('REDIS_ALT'):
+            redis_settings.append(self.config['REDIS_ALT'])
         try:
-            self.redis = await aioredis.create_redis(
-                (redis_settings['host'], redis_settings['port']),
-                db=redis_settings.get('db', 0),
-            )
+            for settings in redis_settings:
+                self.redis_connections.append(
+                    await RedisConnection.create(settings)
+                )
         except (OSError, aioredis.RedisError):
             self.log.exception('could not connect to redis')
             raise
 
-        # Some features (e.g. pinging) don't work on old Redis versions.
-        info = await self.redis.info('server')
-        version_info = info['server']['redis_version'].split('.')
-        major, minor = int(version_info[0]), int(version_info[1])
-        if not (major > 3 or major == 3 and minor >= 2):
-            msg = 'Redis version must be at least 3.2'
-            self.log.exception(msg, version_info=version_info)
-            raise Exception(msg)
```

Comment:
Just to make it explicit, we can safely remove this because we no longer support Redis <= 3.2, yea?

Reply:
Ah, thanks for catching this @thomasst!